src/App.js

```diff
@@ -1,27 +1,10 @@
-import logo from './logo.svg';
+import Calculator from './components/Calculator';
 import './App.css';
```

Comment:
- [ ] You may want to delete this line as deleting all the boilerplate generated with CRA must be removed.